pyproject.toml

```diff
@@ -87,6 +83,9 @@ nanotron = [
   "nanotron",
   "tensorboardX"
 ]
+quality = ["ruff==v0.2.2"]
+tests = ["pytest==7.4.0"]
+dev = ["lighteval[accelerate,quality,tests]"]
```

Comment:
I took a best guess at what the common deps should be here. Usually I would include everything, so let me know if I should also add `tgi` etc

Reply:
Nope, lgtm, cool job :)